src/utils/components/CloudinitModal/CloudInitNetworkForm.tsx

```diff
@@ -54,6 +54,19 @@ export const CloudinitNetworkForm: React.FC<CloudinitNetworkFormProps> = ({
             />
           </FormGroup>
           <FormGroup
+            labelHelp={
+              <HelpTextIcon
+                bodyContent={
+                  <Trans>
```

Comment:
The only thing you're missing is telling the `Trans` component which namespace to use. This can be done by either setting `ns="plugin__kubevirt-plugin"` or `t={t}` on the `Trans` component. 

We're very inconsistent on which to use. You'll see both used in some places. From what I can tell the namespace is enough, so just set `ns="plugin__kubevirt-plugin"`.